src/components/License/sections/LicenseOrganizations.js

```diff
@@ -26,18 +26,19 @@ export default class LicenseOrganizations extends React.Component {
   renderOrgList = (orgs) => {
     return (
       <React.Fragment>
-        { orgs.map((o, index) => (
-          <Layout
-            className="marginTopHalf"
-            data-test-license-org
-            key={index}
-          >
-            {o.org.vendorsUuid ?
-              <Link to={`/vendors/view/${o.org.vendorsUuid}`}>{o.org.name}</Link> :
-              o.org.name
-            }
-            {o.role && `, ${o.role.label}`}
-          </Layout>
+        {orgs.map(o => (
+          o.org ?
+            <Layout
+              className="marginTopHalf"
+              data-test-license-org
+              key={o.org.id}
```

Comment:
While you're in the code, could you tweak this `key` so that it can't be a duplicate? We allow an org to be assigned multiple times with different roles, so `org.id` isn't necessarily unique. `${o.org.id}-${o.role.value}` should be, though.